src/TimePicker/TimePickerDialog.js

```diff
@@ -88,55 +91,49 @@ class TimePickerDialog extends Component {
         color: this.getTheme().clockColor,
       },
       dialogContent: {
-        width: 280,
+        width: 310,
       },
-      body: {
+      dialogBodyContent: {
         padding: 0,
+        minHeight: 434,
+        minWidth: 310,
       },
     };
-
-    const actions = [
-      <FlatButton
-        key={0}
-        label={cancelLabel}
-        primary={true}
-        onTouchTap={this.handleTouchTapCancel}
-      />,
-      <FlatButton
-        key={1}
-        label={okLabel}
-        primary={true}
-        onTouchTap={this.handleTouchTapOK}
-      />,
-    ];
-
     const onClockChangeMinutes = autoOk === true ? this.handleTouchTapOK : undefined;
     const open = this.state.open;
+    const Container = (container === 'inline' ? Popover : Dialog);
 
     return (
-      <Dialog
-        {...other}
-        ref="dialogWindow"
-        style={styles.root}
-        bodyStyle={styles.body}
-        actions={actions}
-        contentStyle={styles.dialogContent}
-        repositionOnUpdate={false}
-        open={open}
-        onRequestClose={this.handleRequestClose}
-      >
-        {open &&
-          <EventListener target="window" onKeyUp={this.handleKeyUp} />
-        }
-        {open &&
-          <Clock
-            ref="clock"
-            format={format}
-            initialTime={initialTime}
-            onChangeMinutes={onClockChangeMinutes}
-          />
-        }
-      </Dialog>
+      <div {...other} ref="root">
```

Comment:
yeah we need it. There was a ClearFix in place of div and the popover was not aligned with the textField. Removing the div does the same. Hence I've kept the div in there.